content/en/_index.md

```diff
@@ -14,75 +13,58 @@ cascade:
-        <a class="btn btn-lg btn-outline-primary mr-3 mb-4" href="https://docs.cobalt.io">
-                API Docs <i class="fab"></i>
-        </a>
-        <a class="btn btn-lg btn-info mr-3 mb-4" href="https://cobalt.io/blog">
-                Blog Posts <i class="fab"></i>
-        </a>
+<div class="jumbotron">
+        <h1>Cobalt Documentation</h1>
+        <p>This is a template for a simple marketing or informational website. It includes a large callout called a jumbotron and three supporting pieces of content. Use it as a starting point to create something more unique.</p>
 </div>
 
-## Support Articles
-
-When customers need help with the Cobalt app, they frequently rely on support articles, available through the [Cobalt Zendesk](https://cobaltio.zendesk.com/hc/en-us/categories/360005476672-Cobalt-Platform) interface.
-
-If you're a Cobalt customer, connect to Zendesk and select the **Sign in** link.
-Once connected, you'll have access to more support articles.
-
-## API Documentation
-
-Cobalt has a [RESTful API](https://docs.cobalt.io)
-that allows you to call the following data related to your pentests:
-
-- Organizations
-- Assets
-- Pentests
-- Findings
-- Events
-- Tokens
-
-To use our REST calls, you'll need an API Token from your
-[Cobalt profile](https://app.cobalt.io/settings/api-token).
-
 
-## Blog Posts
-
-Cobalt has an extensive library of [blog posts](https://cobalt.io/blog),
-designed to help and inform you about:
-
-- Cobalt and our product
-- Profiles for our pentesters
-- Advice for our customers
-- Standards and how you can meet them
-- Life at Cobalt
-
-<!--
-Suggestion from Grahame: I know it's early days, but could be cool to link to a few of these articles. 
-Especially any that are specifically relevant to the API. 
-(Customer testimonials could also be good, in case anyone's coming to the API first.)
--->
-
-{{% pageinfo %}}
-Cobalt is creating product documentation. As we build it, we hope to help you
-visualize how our product can help you simplify the pentest process.
-
-We hope that future docs can help you make best use of Cobalt software.
-{{% /pageinfo %}}
-
-This documentation is a product of Cobalt Labs, Inc. As noted in our
-[Terms of use](https://cobalt.io/terms/general), when we use "Us", "We", "Our", "Cobalt.io",
-or "Cobalt, we're referring to Cobalt Labs, Inc., a Delaware Corporation. 
+<div class="launch-cards">
+      <!-- Example row of columns -->
+      <div class="row">
+        <div class="col-md-4">
+          <img src="gsg/process.png" width="10%">
+          <h2>Heading</h2>
+          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
+          <p><a href="/getting-started/">View details »</a></p>
+        </div>
+        <div class="col-md-4">
+          <img src="gsg/process.png" width="10%">
```

Comment:
`width="10%"` is a "hack." There should be a class for it, but I didn't have time to research.